code/entities/service-bell/ServiceBell.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Editor;
+using Sandbox;
+
+namespace Cinema;
+
+[Library("ent_service_bell"), HammerEntity]
+[Title("Service Bell"), Category("Gameplay"), Icon("notifications")]
+[EditorModel("models/servicebell/servicebell.vmdl")]
+public partial class ServiceBell : AnimatedEntity, ICinemaUse
+{
+
+    /// <summary>
+    /// Set up the model when spawned by the server
+    /// Setup model
+    /// </summary>
+    public override void Spawn()
+    {
+        base.Spawn();
+
+        Transmit = TransmitType.Always;
```

Comment:
Why always?

Reply:
Not sure what to set in this case, just went with what I have seen in other code...  Any suggestions?

Reply:
You can remove it. Only set to Always if there's a specific need for it.

Reply:
Sounds good, I'll remove it